CairoSAC/SACEth001.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:id="SACEth001" xml:lang="en">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Dǝrsāna Giyorgis, Gadla Giyorgis</title>
+                <editor key="DR"/>
+                <editor role="generalEditor" key="AB"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                   This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0.
+               </licence>
+            </availability>
+                
+         </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    
+                    <msIdentifier>
+                        <repository ref="INS1010SAC"/>
+                        <collection>Éthiopien</collection>
+                        <idno>Éthiopien 001</idno>
+                    </msIdentifier>
+                    
```

Comment:
I usually put an empty `<msContent><summary>Dǝrsāna Giyorgis, Gadla Giyorgis</summary></msContent>` also in preliminary stubs